`dora check --dataflow` provides ambiguous "INFO" messages for missing files

When running dora check --dataflow <FILE.yml>, if the dataflow references non-existent python/rust source files, the CLI behavior is confusing and misleading.

It prints a generic system message: INFO: Could not find files for the given pattern(s). This message does not specify which file is missing or which node is affected.

Crucially, it still reports success for the Coordinator/Daemon checks (ok), leading the user to believe the dataflow itself is valid.

### **Steps to Reproduce**
Create a broken.yml with a node referencing a missing file:

YAML
```
nodes:
  - id: ghost_node
    path: missing_file.py
```
**Run the check:**

Bash
```
dora check --dataflow broken.yml
```

### **Current Behavior**
```
Dora Coordinator: ok
Dora Daemon: ok
C:\test>INFO: Could not find files for the given pattern(s).
```

<img width="483" height="161" alt="Image" src="https://github.com/user-attachments/assets/8c9e6bdb-5976-4166-a929-11ccb46c1e08" />

### **Expected Behavior**
The CLI should explicitly validate the path of each node. If a file is missing, it should:
Print a clear [ERROR] message identifying the specific node ID and the missing path.
Exit with a non-zero status code to indicate failure.
(Optional) List all missing files in one go rather than stopping at the first one.

### **Environment**
OS: Windows (confirmed), likely Linux/macOS too.
Dora Version: v0.3...

--- a/libraries/core/src/descriptor/mod.rs
+++ b/libraries/core/src/descriptor/mod.rs
@@ -10,7 +10,6 @@ use std::{
     path::{Path, PathBuf},
     process::Stdio,
 };
-use tokio::process::Command;
 
 // reexport for compatibility
 pub use dora_message::descriptor::{
@@ -224,14 +223,21 @@ pub fn resolve_path(source: &str, working_dir: &Path) -> Result<PathBuf> {
     } else if which::which("uv").is_ok() {
         // spawn: uv run which <path>
         let which = if cfg!(windows) { "where" } else { "which" };
-        let _output = Command::new("uv")
+        let output = std::process::Command::new("uv")
             .arg("run")
             .arg(which)
             .arg(&path)
             .stdout(Stdio::null())
-            .spawn()
-            .context("Could not find binary within uv")?;
-        Ok(path)
+            .stderr(Stdio::null())
+            .output()
+            .context("Could not run `uv run` to find binary")?;
+        if output.status.success() {
+            Ok(path)
+        } else if let Ok(abs_path) = which::which(&path) {
+            Ok(abs_path)
+        } else {
+            bail!("Could not find source path {}", path.display())
+        }
     } else if let Ok(abs_path) = which::which(&path) {
         Ok(abs_path)
     } else {

